client/dfcache/dfcache.go

```diff
@@ -0,0 +1,293 @@
+/*
+ *     Copyright 2022 The Dragonfly Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dfcache
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"d7y.io/dragonfly/v2/client/config"
+	logger "d7y.io/dragonfly/v2/internal/dflog"
+	"d7y.io/dragonfly/v2/pkg/basic"
+	"d7y.io/dragonfly/v2/pkg/rpc/base"
+	"d7y.io/dragonfly/v2/pkg/rpc/dfdaemon"
+	daemonclient "d7y.io/dragonfly/v2/pkg/rpc/dfdaemon/client"
+)
+
+// Format that's used to cast the given cid to URI.
+// "d7y" as scheme, and absulute path starts with "/"
+const cidURIFormat = "d7y:/%s"
```

Comment:
`cidURIFormat` should start with `d7y://`(two slash)
Refer: https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Example_URIs

Reply:
According to the URI syntax, https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax

URI = scheme ":" ["//" authority] path ["?" query] ["#" fragment]

"d7y" is the scheme part, and "// authority" is optional, and "path" is required, and absolute path begins with "/", so I used "d7y:/%s" format, as "scheme:path", the authority part is skipped.

Reply:
Okay, keep it "d7y:/"